db/kvdb/kvdb.go

```diff
@@ -49,6 +49,8 @@ type KVDB struct {
 	CurrentIdx   common.Idx
 	CurrentBatch common.BatchNum
 	m            sync.Mutex
+	mutexDelOld  sync.Mutex
```

Comment:
Now we have 2 mutexes. Can you please give a better name for another one? How do I know the mutex I must use?

Reply:
Ok

Reply:
Done